docs/advanced-topics/checksum-support.rst

```diff
@@ -19,6 +19,19 @@ More specifically, the endpoints that contain checksums for files are as follows
 
 The id parameter used in the endpoints above can be found on an entry's public page; underneath the Info tab, look for the bolded words **TRS**.
 
+After gathering the checksum using the above method you can verify a descriptor's checksum using the sha1sum terminal application.
```

Comment:
On a Mac, by default, it looks like it's `shasum`. I ran your commands below, and it seems to work with it.

```
curl -s https://dockstore.org/api/ga4gh/trs/v2/tools/$trsid/versions/$version/PLAIN-WDL/descriptor | shasum
c8ea6c657fd2e2ca9743567331e0947aa3eed33a  -
```

Does the trailing dash matter? I guess this is a visual inspection anyway.

But in any case, suggest mentioning shasum and Mac.

Reply:
The trailing dash is from the shasum application and it just prints the filename. Since we are piping stdout it just puts a dash. shasum and sha1sum do the same thing on mac and linux so I'll just use shasum. shasum defaults to sha1